app/controllers/administrative-units/administrative-unit/core-data/index.js

```diff
@@ -0,0 +1,29 @@
+import { helper } from '@ember/component/helper';
+import Controller from '@ember/controller';
+import { ID_NAME } from 'frontend-contact-hub/models/identifier';
+import WorshipServiceModel from 'frontend-contact-hub/models/worship-service';
+
+const SHAREPOINT_LINK_BASE = {
+  WORSHIP_SERVICE:
```

Comment:
Hmm, I'm not sure about that. These URLs won't change depending on the environment, right? So I don't really see the added value of making this configurable at runtime (besides not hardcoding the URL, of course).

I'll check with Yassin and act accordingly?

Reply:
well, if they change the url on sharepoint (dunno, if they move the file or smth), we have to release a new version of the frontend just to change the url; that's what i had in mind, but can stay like that and be refined in the future

Reply:
I discussed this with Yassin and he thinks that we shouldn't spend time on making it configurable since it is unlikely that the URLs will ever change.

So I think this can be merged if there are no other remarks :+1:.